packages/flame/lib/src/components/isometric_tile_map_component.dart

```diff
@@ -75,20 +82,53 @@ class IsometricTileMapComponent extends PositionComponent {
   /// This is the size the tiles will be drawn (either original or overwritten).
   Vector2 get effectiveTileSize => destTileSize ?? tileset.srcSize;
 
-  /// This is the vertical height of each block; by default it's half the tile
-  /// size.
+  /// The current scaling factor for the isometric view.
+  double get scalingFactor => effectiveTileSize.y / effectiveTileSize.x;
+
+  /// This is the vertical height of each block; by default it's half the tile size.
   double get effectiveTileHeight => tileHeight ?? (effectiveTileSize.y / 2);
 
+  /// gets the effective propSize. PropSize should be assigned if prop dimension are different from srcSize of tilset
+  Vector2 get propsMaxSize => propSize ?? effectiveTileSize;
+
+  // render specific variables
+
+  /// sourceRenderPosition of the sprite on the image
+  final Vector2 _renderSourcePosition = Vector2.zero();
+
+  /// renderSprite
```

Comment:
Private variables dont really have to be documented, especially if it is just the name itsef ;-) no extra value in that

Reply:
yup This just slipped through.